Tests/Unit/MSFT_WindowsOptionalFeature.Tests.ps1

```diff
@@ -317,6 +317,10 @@ try
                         $propertiesAsStrings.Contains("Name = Object $objectNumber, Value = Value $objectNumber, Path = Path $objectNumber") | Should -BeTrue
                     }
                 }
+
+                It 'Should return an empty array and not a null object when input is empty' {
```

Comment:
Can say `It 'Should return an empty array and not a null object when input is empty'`?

Reply:
I believe it does say that, no?

Reply:
I meant the input should say "when input is empty array".

Reply:
Updated to:
`'Should return an empty array and not a null object when input is an empty array'`